recipes/metasnv/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include"
+export CPPFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+tag=share/metasnv-1.0.1/
+odir=$PREFIX/$tag
+
+make
+mkdir -p $odir
+cp -pr src $odir
+cp -pr metaSNV.py $odir
+cp -pr metaSNV_post.py $odir
+
+cat >$PREFIX/bin/metaSNV_post.py <<EOF
```

Comment:
why not put the original `metaSNV*.py` in `bin/` ?

Reply:
Because the code expects that everything will be located relative to it.

So, it's either this or patching the code (which I don't want to go into).

Reply:
would a symbolic link from odir to bin work ?

Reply:
No. I tried that. The ARGV[0] path is still set to the file on bin and then the package fails to find its other files.

Reply:
alright !